### PR TITLE
Use /healthz/ready when verifying API

### PR DESCRIPTION
**Pull request by abutcher**
@thoraxe

### CODE REVIEW

**Comment by detiber** [comment]
@abutcher I'm a but confused...  We currently verify the api server is up before leaving the master role, shouldn't that be sufficient for this?

**Comment by abutcher** [comment]
@detiber The problem might be that this check isn't sufficient for determining if the API has completely started? @thoraxe and I are troubleshooting an issue where his nodes don't start on the first run.

**Comment by abutcher** [comment]
@detiber I'm going to change the master health check to use `/healthz/ready`

**Comment by abutcher** [comment]
aos-ci-test

**Comment by detiber** [comment]
assuming tests pass, :+1: from me.

**Comment by abutcher** [comment]
aos-ci-test

File: roles/openshift_node/tasks/main.yml [hunk 1]
```diff
@@ -124,9 +124,9 @@
   # Using curl here since the uri module requires python-httplib2 and
   # wait_for port doesn't provide health information.
   command: >
-    curl -k --head --silent {{ openshift_node_master_api_url }}
+    curl -k --silent {{ openshift_node_master_api_url }}/healthz/ready
   register: api_available_output
-  until: api_available_output.stdout.find("200 OK") != -1
+  until: api_available_output.stdout == 'ok
```
Comment (brenton) [inline roles/openshift_node/tasks/main.yml]: missing a ' here.

Comment (abutcher) [inline roles/openshift_node/tasks/main.yml]: doh

**Comment by abutcher** [comment]
@brenton quotes fixed

**Pull request merged**